src/ext.js

```diff
@@ -153,6 +153,33 @@ export default function ext({ translator }) {
                   defaultValue: false,
                   show: (data) => checkShowAction(data, 'partial'),
                 },
+                // adds automation to actions and adds a dropdown property panel
+                // item to select the automation for the button to trigger
+                automation: {
+                  type: 'string',
+                  component: 'dropdown',
+                  label: 'Select automation',
+                  ref: 'automation',
+                  options: async () => {
+                    const automations = await fetch('../api/v1/items?resourceType=automation').then((response) =>
```

Comment:
We should probably have a look at the usage of fetch. In client we make use of axios. Easy enough to add though. 
The url used should probably be better formed or maybe even as input from the user. This will only work if the button is clicked and configured on the same domain as the automation is running. Pretty sure that is not always the case. 

@Caele Do you know of any restrictions on fetch versus axios? I know that for axios we add the correct headers so probably best to use axios here as well